app/src/main/java/com/android/doctorapp/ui/userdashboard/userfragment/BookingDetailViewModel.kt

```diff
@@ -54,8 +58,7 @@ class BookingDetailViewModel @Inject constructor(
                         fireStore
                     )) {
                     is ApiSuccessResponse -> {
-                        setShowProgress(false)
-                        _navigationListener.value = true
+                        sendRejectedNotification("Appointment rejected by")
```

Comment:
Take this message in constant file because if in future need to change then only one place changed.